application.go

```diff
@@ -149,6 +151,12 @@ type Stats struct {
 	LifeTime map[string]float64 `json:"lifeTime"`
 }
 
+// Secret is the environment variable and secret store path associated with a secret.
```

Comment:
Maybe add another quick sentence here too saying something about `EnvVar` being populated from the regular environment variables.